server/fusor.gemspec

```diff
@@ -13,4 +13,6 @@ Gem::Specification.new do |s|
   s.files      = Dir["{app,config,db,lib}/**/*"] + ["LICENSE", "Rakefile", "README.md"]
   s.test_files = Dir["test/**/*"]
 
+  s.add_dependency "active_model_serializers", '~> 0.9'
+
```

Comment:
I prefer `active_model_serializers` for side-loading, something that isn't trivial in rabl but very simple out-of-the-box in active_model_serializers